torch/_functorch/_aot_autograd/traced_function_transforms.py

```diff
@@ -247,7 +247,8 @@ def inner_fn(primals: List[Any], tangents: List[Any]):
         ]
 
     def inner_fn_with_anomaly(*args):
-        with fx_traceback.preserve_node_meta(), warnings.catch_warnings():
+        with fx_traceback.preserve_node_meta(), warnings.catch_warnings(), \
+                FunctionalTensorMode(aot_config.pre_dispatch):
```

Comment:
Since we need the `FunctionalTensorMode` in two places - do you mind adding a quick `Note [Accessing .grad_fn on FunctionalTensor]` comment and mentioning it both places so we have a nice paper trail?

Reply:
Yep sure thing!